src/components/Certify/Certify.vue

```diff
@@ -109,11 +110,7 @@ export default class Certify extends Vue {
   font-size: 0.875rem;
 }
 
-.certify-container {
-  padding: 1.25rem;
-}
-
-.certifiedby-container {
+.certified-by-container {
```

Comment:
I no longer see this class used in the template. (Also in the media query block below.)